compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5839,16 +5844,22 @@ TR::Register * OMR::Power::TreeEvaluator::ibyteswapEvaluator(TR::Node *node, TR:
          {
          generateTrg1Src1Instruction(cg, TR::InstOpCode::brw, node, tgtRegister, srcRegister);
          }
+      else if (cg->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P8))
```

Comment:
Can you please add comments explaining why `rlwimi` isn't used prior to P8?

Reply:
Commented.